edit/workload/index.vue

```diff
@@ -595,6 +595,15 @@ export default {
       }
     },
 
+    cleanUpServices() {
+      (this.servicesOwned || []).forEach((svc) => {
+        try {
+          this.$store.dispatch('cluster/find', { type: SERVICE, id: `${ svc.metadata.namespace }/${ svc.metadata.name }` }).then((svc) => {
+            svc.remove();
+          });
+        } catch {}
+      });
+    }
```

Comment:
Ditto on the annotation to make sure we made it thing.

Seems like this should be moved to the workload model and also be triggered on a regular delete of a workload to cleanup the services we made referencing it (I believe the Norman API does that for the ones it made?)